en/docs/get-started/install.md

```diff
@@ -1,4 +1,4 @@
-# Install Ballerina plugin
+# Install Ballerina extension
```

Comment:
Shall we say `Install the Ballerina extension`

No need to drop articles IMO

Reply:
Fixed.